content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,329 @@
+
+Using the continuity of the tangential \\(\\mathbf{E_x}\\) and \\(\\mathbf{H_y}\\) field at the interfaces, we find an iterative relation between the fields in consecutive layers:
+
+.. math::
+    \left(\begin{matrix} E_{x,j} \\ H_{y,j} \end{matrix} \right) = P_j T_j P^{-1}_J \left(\begin{matrix} E_{x,j+1} \\ H_{y,j+1} \end{matrix} \right)
+
+We are now only missing a Boundary Condition to be able to compute our MT forward modeling. A reasonable one is to set the Down Amplitude to 1 and the Up Amplitude to 0 in the last layer, as there is no reflection from an other interface below.
+
+.. math::
+    \left(\begin{matrix} U_n \\ D_n \end{matrix} \right)  = \left(\begin{matrix} 0 \\ 1 \end{matrix} \right) 
+
+Building Intuition for MT problems
+----------------------------------
+
+Skin Depth and Depth of investigation
+*************************************
+
+ .. figure:: images/SkinDepth_MT.png
+    :align: right
+    :scale: 50% 
+    :name: SkinDepth_MT
+
+    Depth of investigation in MT
+
+Take the amplitude of the incident component of the electric wave, :math:`E_{x} (z) =  D e^{Im(k) z}`.
+
+The skin depth :math:`\delta` is defined as the depth where the signal has decayed to a factor :math:`\frac{1}{e}(\simeq` 36%).
+ 
+.. math::
+    e^{-i Im(k) \delta} = \frac{1}{e}
+ 
+.. math::
+    \delta = \sqrt{ \frac{2}{\omega \mu \sigma}} \simeq \frac{500}{\sqrt{\sigma f}}
+    :label: Skin Depth
+
+We see the skin depth is highly dependent on both the frequency of our signal and the conductivity of the Earth material. In air , the conductivity is almost 0, so we do not notice important decreased of the electromagnetic wave. In the ground, this is different.
+ - the more conductive, the faster the decay is. MT can see very deep in resistive environment.
+ - The lower the frequency, the slower the decay is. Lowest frequencies sample the deepest structures while high frequencies bring information on shallower structures.
+
+ In :numref:`SkinDepth_MT` and in the movie, we can see that even at very high frequency (20000 Hz), MT is still a deep exploration method in resistive environment (:math:`10^{-5} S/m`) with a skin depth of about 1125m. Skin Depth is often use as an estimator for the depth of investigation of a survey.
+
+Reflection and Transmission Coefficients
+****************************************
+
+.. figure:: images/Reflection_MT.png
+ :align: right
+ :scale: 50% 
+ :name: Reflection_MT
+
+ Reflection at 
+ interface
+
+.. figure :: images/Reflection_Efield.png 
+ :align: right
+ :scale: 50% 
+ :name: Reflection_Efield
+
+ Notations for 
+ reflection
+
+
+
+Let define at the j-th interface :math:`(E^i ; H^i)` as the incident waves, :math:`(E^r ; H^r)` as the refleted wave and :math:`(E^t ; H^t)` as the transmitted wave into the ground (:numref:`Reflection_Efield`) 
+
+Using the :ref:`interface conditions<interface_conditions>` for the tangential components of the electric, we can write:
+
+.. math::
+    E^i + E^r = E^t
```

Comment:
I am looking into how to fix these numbers... odd, but we can merge it even without these fixed (we may switch to a different theme that might be better for these sorts of things)